CHANGELOG.md

```diff
@@ -163,6 +163,8 @@
 
 ### Dependencies
 
+- Bumped OpenResty from 1.21.4.1 to 1.21.4.2
```

Comment:
The change log entry should be added in a new  `unreleased` section after 3.4 release.

Reply:
@chronolaw, I agree on that, thus I guess we need to wait for release, a merge to changelog from release, and then add a new section. Or we just continue normally like here, and resolve this issue when we merge changelog from 3.4 to master.